src/components/POLine/POLineForm.js

```diff
@@ -203,8 +203,12 @@ function POLineForm({
     : <FormattedMessage id="ui-orders.line.paneTitle.new" />;
   const paneFooter = getPaneFooter();
 
-  const changeLocation = (location, fieldName) => {
-    change(fieldName, location.id);
+  const changeLocation = (location, locationFieldName, holdingFieldName, holdingId) => {
+    change(locationFieldName, location?.id || location);
+
+    if (holdingFieldName) {
+      change(holdingFieldName, holdingId);
+    }
   };
```

Comment:
Doesn't look clear to my taste:
- name is `changeLocation`, but we're potentially changing Holdings too
- `location` argument can be an object with ID field or the ID itself
- arguments order is not consistent (id, name, name, id) - maybe use an object as argument?
- the logic is repeated in OrderTemplatesEditor.js

Corresponding usage - `onChange` in `<FieldHolding>` - feels weird accordingly

Reply:
because it's still location, but specified with holding